telegram/_files/inputmedia.py

```diff
@@ -602,5 +698,19 @@ def __init__(
             api_kwargs=api_kwargs,
         )
         with self._unfrozen():
-            self.thumb: Optional[Union[str, InputFile]] = self._parse_thumb_input(thumb)
+            self.thumbnail: Optional[Union[str, InputFile]] = self._parse_thumb_input(thumbnail)
             self.disable_content_type_detection: Optional[bool] = disable_content_type_detection
+
+    @property
+    def thumb(self) -> Optional[Union[str, InputFile]]:
+        """:class:`telegram.InputFile`: Optional. |thumbdocstringbase|
+
+        .. deprecated:: NEXT.VERSION
+           |thumbattributedeprecation| :attr:`thumbnail`.
+        """
+        warn(
+            "Bot API 6.6 renamed the attribute 'thumb' to 'thumbnail'.",
+            PTBDeprecationWarning,
+            stacklevel=2,
+        )
```

Comment:
sorry if this was already asked before, but can't we just use the warning helper function for attribute from `warnings_transition.py`?

Reply:
yes, I must have forgotten to replace it. Thank you! Done